src/sentry/incidents/endpoints/organization_alert_rule_details.py

```diff
@@ -127,9 +127,10 @@ def fetch_alert_rule(request: Request, organization, alert_rule):
         if request.user.id == rule_snooze.owner_id:
             serialized_rule["snoozeCreatedBy"] = "You"
         else:
-            user = user_service.get_user(rule_snooze.owner_id)
-            if user:
-                serialized_rule["snoozeCreatedBy"] = user.get_display_name()
+            if rule_snooze.owner_id:
+                user = user_service.get_user(rule_snooze.owner_id)
+                if user:
+                    serialized_rule["snoozeCreatedBy"] = user.get_display_name()
```

Comment:
How is this displayed to the user if `["snoozeCreatedBy"]` is not set?

Reply:
the display name is always something https://github.com/getsentry/sentry/blob/b4e4025237f04fb14ed9acce14da8f6aab874b6a/src/sentry/models/user.py#L261-L262

Reply:
Yeah I meant when user is None. It seems ok since that was there before your change too.